src/fairseq2/nn/transformer/multihead_attention.py

```diff
@@ -187,6 +187,7 @@ def __init__(
         model_dim: int,
         num_heads: int,
         *,
+        input_dim: Optional[int] = None,
```

Comment:
I don't think `input_dim` is the right name for this parameter. Pedantically "input" to MHA is the query. K/V values are more like the "state dictionary" to attend against. How about naming this `kv_model_dim` or `kv_dim`?

Reply:
`kv_dim` sounds just right!